verifiers/treeshaking/src/functions/external.functions.ts

```
@@ -0,0 +1,19 @@
+ */
+
+import { pikkuSessionlessFunc } from '../../.pikku/pikku-types.gen.js'
+
+export type TestExternalInput = { value: string }
+export type TestExternalOutput = { result: string }
+
+export const testExternal = pikkuSessionlessFunc<
+  TestExternalInput,
+  TestExternalOutput
+>({
+  func: async ({}, data, { rpc }) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)